Rakefile

```
@@ -16,3 +17,19 @@ task :rubocop do
 end
 
 task default: %i[test rubocop]
+
+namespace :healthcards do
+  desc 'Generates a random DID and prints out payload.'
+  task :did do
+    include Dids
+    include Keys
+    enc_key = Keys.generate_key
+    sign_key = Keys.generate_key
+    update_key = Keys.generate_key
+    rec_key = Keys.generate_key
+    did = Dids.generate_did(enc_key[:publicJwk], sign_key[:publicJwk], update_key[:publicJwk], rec_key[:publicJwk])
```

Typically when including a module you'll just reference the module methods directly without prepending the module. In a regular class something like this shouldn't work, unless the methods are defined as class methods (not sure why it works here though...maybe rake does something special that I don't understand)
```suggestion
    enc_key = generate_key
    sign_key = generate_key
    update_key = generate_key
    rec_key = generate_key
    did = generate_did(enc_key[:publicJwk], sign_key[:publicJwk], update_key[:publicJwk], rec_key[:publicJwk])
```